lexicons/com/atproto/admin/defs.json

```diff
@@ -587,6 +587,23 @@
         }
       }
     },
+    "modEventFlag": {
+      "type": "object",
+      "description": "Add/Remove a flag on a subject",
+      "required": ["add", "remove"],
```

Comment:
do the arrays need to be required, or can they be omitted if empty?

Reply:
I was trying to stay in line with the labelling lexicons where we require both `create` and `negate` properties. I don't think we allow defining required fields as "one or the other" which is probably why this is kinda needed?